modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+:FeatureName: vSphere topology Awareness
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Procedure
+To deploy {product-title} for vSphere storage on different zones and regions:
```

Comment:
Regarding the colon at the end here: Use a colon after the introduction to a vertical list, including a procedure or substeps in a procedure. To avoid translation problems, make the introduction a complete sentence. Note that I only think this applies to the introduction sentence, not the following (i.e., 1., 2., etc.). 

See: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons

Could we reword this as something like: 

Use the following procedure to deploy (product-title} for vSphere storage on different zones and regions: